lib/datadog/appsec/contrib/rack/gateway/watcher.rb

```diff
@@ -21,6 +21,7 @@ def watch
                 watch_request(gateway)
                 watch_response(gateway)
                 watch_request_body(gateway)
+                watch_user_id(gateway)
```

Comment:
It seems to make sense to have this set up call done in that `contrib/datadog` integration I suggested. I'm still not sure it shoudl be in `contrib` or if it should be some internal integration of sorts, but I really don't feel like it belongs to `contrib/rack`.